content/en/users/data/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Data Services
+linkTitle: Data
+type: docs
+weight: 40
+description: >
+  Data services in the EGI Cloud
+---
+
+## Working with data
+
+One of the main daily activities of researchers is to collect or create data
+that needs to be processed, analyzed, or shared. The EGI offering in this area
+is quite vast and includes both low level
+[storage services](./storage/) and
+[data management](./management/) frameworks, as well as specialized
+[data transfer](./management/data-transfer/) services.
```

Comment:
Should we move data-transfer to stand outside of data management to keep this logic? @andrea-manzi

Reply:
i would keep like this now, maybe to revisit when also openrdm doc is in

Reply:
Added content for the openRDM page